driver-core/src/test/functional/com/mongodb/internal/operation/FindOperationSpecification.groovy

```diff
@@ -480,6 +482,38 @@ class FindOperationSpecification extends OperationFunctionalSpecification {
         async << [true, false]
     }
 
+    @IgnoreIf({ !serverVersionAtLeast(4, 3) || !ClusterFixture.isDiscoverableReplicaSet() })
```

Comment:
Hedging is only supported on sharded clusters, so we should also test against a sharded cluster.  Just ignore if it's a standalone